src/containers/Tenant/Diagnostics/TopQueries/getTopQueriesColumns.tsx

```diff
@@ -66,6 +72,23 @@ const userSIDColumn: Column<KeyValueRow> = {
     align: DataTable.LEFT,
 };
 
+const oneLineQueryTextColumn: Column<KeyValueRow> = {
+    name: TOP_QUERIES_COLUMNS_IDS.OneLineQueryText,
+    header: 'QueryText',
+    render: ({row}) => <OneLineQueryWithPopover value={row.QueryText?.toString()} />,
+    sortable: false,
+};
+
+const queryHashColumn: Column<KeyValueRow> = {
+    name: TOP_QUERIES_COLUMNS_IDS.QueryHash,
+    render: ({row}) =>
+        // We use unsigned right shift operator (>>>) to avoid negative values
+        // eslint-disable-next-line no-bitwise
+        (crc32.str(String(row.QueryText)) >>> 0).toString(16).toUpperCase().padStart(8, '0'),
```

Comment:
Can you please make it a separate function in `utils`? And could you please split it to steps and explain every step in comments
1. crc32
2. `>>>>`
3. toString(16)
4. toUpperCase()
5. padStart(8, '0')